File: Batch-Compliance/python/upload_ids.py
import requests

# Replace with your job download_url
upload_url = ''

# Replace with your file path that contains the list of Tweet IDs or User IDs, one ID per line
file_path = ''

headers = {'Content-Type': "text/plain"}


def connect_to_endpoint(url):
    response = requests.put(url, data=open(file_path, 'rb'), headers=headers)
    print(response.status_code)
    if response.status_code != 200:
        raise Exception(response.status_code, response.text)
    return response.text


def main():
    response = connect_to_endpoint(upload_url)
    print(response)


if __name__ == "__main__":
    main()

File: Batch-Compliance/js/upload_ids.js
const got = require('got');
const fs = require('fs');

// Replace with your job download_url
uploadUrl = ''

// Replace with your file path that contains the list of Tweet IDs or User IDs, one ID per line
const readStream = fs.createReadStream('/path/to/file');

async function getRequest() {

    const res = await got.put(uploadUrl, {
        body: readStream,
        headers: {
            "Content-Type": "text/plain"
        }
    })

    if (res.statusCode == 200) {
        return res.statusCode;
    } else {
        throw new Error('Unsuccessful request');
    }
}

(async () => {

    try {
        // Make request
        const response = await getRequest();
        console.dir(response, {
            depth: null
        });

    } catch (e) {
        console.log(e);
        process.exit(-1);
    }
    process.exit();
})();
